src/UI/Implementation/Component/Input/Field/Renderer.php

```diff
@@ -557,6 +552,24 @@ protected function renderRadioField(Component\Input\Field\Radio $input, Renderer
 		return $tpl->get();
 	}
 
+
+	protected function getOptionalGroupOnLoadCode($id) {
+		return <<<JS
+var $id = $("#$id");
+var {$id}_group = $id.siblings(".form-group").show();
+var {$id}_adjust = function() {
+	if ({$id}[0].checked) {
+		{$id}_group.show();
+	}
+	else {
+		{$id}_group.hide()
+	}
+}
+$id.change({$id}_adjust);
+{$id}_adjust();
+JS;
```

Comment:
Maybe better externalize this JS-block in some js-file? Input/Field/input.js?

Reply:
I gave this some thought and came to the following conclusion:

* Adding a new `optional_group.js` is bad, because we will load it very often (for every form) but need it very little (only when an optional group exists).
* `input.js` seems to include stuff for _all_ inputs, not for specific ones. Adding specific stuff is odd.
* Cleaning up the javascript for inputs (and in general) is certainly require but out of scope.

So I just added it inline.